### PR TITLE
A documentation page for explaining Secrets for the average user.

### PR DESCRIPTION
**Pull request by mactsouk**

### CODE REVIEW

**Comment by darkofabijan** [comment]
@mactsouk one general note here. Don't combine `secret` with `bucket`. Just say `secret`.

**Comment by mactsouk** [comment]
@darkofabijan will do.

File: using-secrets_5b8506120428631d7a8a7239.md [hunk 1]
```diff
@@ -387,17 +413,69 @@ file of that secret and execute `sem create`.
...
+	    os_image: ubuntu1804
+    
+	blocks:
+	  - name: Using SSH key files
+	    task:
+	      jobs:
+	      - name: Using server_key
+	        commands:
+	          - ls -l ~/.ssh
+			  - chmod 600 ~/.ssh/server_key
```
Comment (shiroyasha) [inline using-secrets_5b8506120428631d7a8a7239.md]: By looking at this, I might even argue that you should be able to define the file mode in the secret. cc @mactsouk @darkofabijan let me know what you think about that.

Comment (mactsouk) [reply using-secrets_5b8506120428631d7a8a7239.md]: It is not a bad idea - or maybe keep the file permissions of the original file.
On the other hand, using `chmod` is not a big issue.

**Pull request merged**